x-pack/plugins/alerting/server/routes/get_rule_execution_log.ts

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { IRouter } from 'kibana/server';
+import { schema } from '@kbn/config-schema';
+import { ILicenseState } from '../lib';
+import { GetExecutionLogByIdParams } from '../rules_client';
+import { RewriteRequestCase, verifyAccessAndContext } from './lib';
+import { AlertingRequestHandlerContext, INTERNAL_BASE_ALERTING_API_PATH } from '../types';
+
+const paramSchema = schema.object({
+  id: schema.string(),
+});
+
+const querySchema = schema.object({
+  date_start: schema.string(),
+  date_end: schema.maybe(schema.string()),
+  filter: schema.maybe(schema.string()),
+  per_page: schema.number({ defaultValue: 10, min: 1 }),
+  page: schema.number({ defaultValue: 1, min: 1 }),
+  sort_field: schema.oneOf([schema.literal('timestamp'), schema.literal('duration')], {
```

Comment:
Also for Sorting, can we just use the same type than elasticsearch `estypes.Sort` from `import * as estypes from '@elastic/elasticsearch/lib/api/typesWithBodyKey';`

Reply:
@XavierM Updated in this commit: https://github.com/elastic/kibana/pull/127339/commits/8636b8f551820ddb2ae9e11f5dd41676a3fd5d39

WDYT?

Reply:
beautiful, thank you!!!